docs/website/webpack.config.js

```diff
@@ -4,7 +4,9 @@ module.exports = {
   mode: 'development',
   devtool: 'source-map',
   entry: {
-    'localdev': './src/localdev.ts',
+    'site': './src/site.ts',
+    'polyfill-light': './src/polyfill-light.ts',
+    'polyfill-medium': './src/polyfill-medium.ts'
```

Comment:
we don't need `polyfill-full` support?

Reply:
We do not support IE11 and ES5-time browsers

Reply:
this is great news!